Makefile

```diff
@@ -0,0 +1,8 @@
+install:
+	pip install pip-tools
+	pip-compile dev-requirements.in
+	pip-compile requirements.in
+	pip-sync requirements.txt dev-requirements.txt
```

Comment:
this is not required and misses the point of pip-tools
see the dockerfile for how to install production packages or the gitlab file for how to install dev packages